packages/flutter/test/painting/text_span_test.dart

```diff
@@ -59,7 +57,6 @@ void main() {
       '    "b"\n'
       '    TextSpan:\n'
       '      (empty)\n'
-      '  <null child>\n'
```

Comment:
is the code that generates this string also removed?

Reply:
No, but it does need an ignore on dead code:
 https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/painting/text_span.dart#L372